index.html

```diff
@@ -1,20 +1,320 @@
+                Hevons Farms
+              </a>
+            </div>
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Products</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Services</a>
+              </li>
+            </ul>
+            <button
+              class="btn btn-outline-warning brand-btn ms-lg-5"
+              type="submit"
+            >
+              Partner
+            </button>
+          </div>
+        </div>
+      </nav>
+      <nav
+        class="bg-light container-fluid d-flex pt-2 px-5 d-none d-md-flex justify-content-end"
+        aria-label="breadcrumb"
+      >
+        <ol class="breadcrumb breadcrumb-space">
+          <li class="breadcrumb-item">
+            <a href="#"><span class="material-icons"> home_filled </span></a>
+          </li>
+          <li class="breadcrumb-item active text-primary" aria-current="page">
+            Library
+          </li>
+        </ol>
+      </nav>
+    </header>
+    <main class="bg-light">
+      <!-- Banner section -->
+      <section class="px-5 banner">
+        <h1 class="display-1 fw-bolder text-capitalize intro-txt">
+          Sustaining the future with food security today
+        </h1>
+        <div class="shadow-lg py-1 my-4 col-sm-12 col-md-12 col-lg-8">
+          <p class="fs-6 fw-light desc-txt my-1 mx-1 p-2">
+            Many countries are facing the double burden of hunger and
+            undernutrition alongside overweight and obesity, with one in three
+            people across the globe currently suffering from some form of
+            malnutrition. Indeed it is not unusual to find people with different
+            forms of malnutrition living side-by-side in one country, in one
+            community, or even in the same household.
+          </p>
+        </div>
+        <a
+          href="#"
+          class="display-3 text-capitalize text-decoration-none fs-1 fw-bold"
+          >join us today!</a
+        >
+        <p class="display-5 fs-5">
+          Together, let's secure a future for our generations yet to come
+        </p>
+      </section>
+      <section class="bg-dark d-flex flex-column gap-4 services px-sm-2 py-3">
+        <h3 class="h3 text-center">Our Services</h3>
+        <div class="line"></div>
+        <ul
+          class="d-md-flex d-block gap-1 flex-grow-1 flex-wrap justify-content-center"
+        >
+          <li class="d-flex flex-row flex-sm-column gap-2 py-2">
+            <img
+              src="./img/cattle.jpeg"
+              alt="cattle-images"
+              class="rounded-circle"
+            />
+            <p class="h5 fw-bold text-center inner-heading">Livestock Farms</p>
```

Comment:
- [ ] Kindly add the padding in this section as well as the hover effect to all your cards. This will improve the UI of your site 💯 
- [ ] Also kindly use the correct colors for all your heading. For example in this section, the heading should be white but yours is red. Kindly rectify this issue throughout your site 👍 
- Your design:
![Capture3](https://user-images.githubusercontent.com/85113567/181269089-d587f908-ecee-484f-a2ec-ba06f5fccf8f.PNG)
- Required design:
![Capture4](https://user-images.githubusercontent.com/85113567/181269138-5904416e-b700-4fed-9f8a-95956161084c.PNG)

Reply:
Resolved